test/unit/foreman_fog_proxmox/helpers/proxmox_container_helper_test.rb

```diff
@@ -183,6 +220,15 @@ class ProxmoxContainerHelperTest < ActiveSupport::TestCase
           nics[0][:net0]
       end
 
+      test '#interface with name eth0 and bridge in host edit' do
+        deletes = []
+        nics = []
+        add_or_delete_typed_interface(host_edit['interfaces_attributes']['0'], deletes, nics, type)
+        assert 1, nics.length
+        assert nics[0].key?(:net0)
+        assert_equal 'hwaddr=8E:96:2A:D3:42:80,bridge=vmbr0,name=eth0,ip=dhcp,firewall=0', nics[0][:net0]
```

Comment:
The test will fail until fog-proxmox PR is merged, https://github.com/fog/fog-proxmox/pull/94